ovs-p4rt/sidecar/testing/p4Info.txt

```diff
@@ -0,0 +1,2760 @@
+pkg_info {
```

Comment:
Since this file is a compiler-generated artifact, this file will need to get periodically updated with new compiler versions (frequency of compiler version change is 2-3x per month). 

It would be a good idea to add some metadata to indicate which p4c compiler generated this file (ideally outside of this file, so we can simply drop the p4Info.txt file without editing it).

Reply:
Another change that could trigger this is the p4 program file change itself. I am not aware of versioning of individual p4 programs - perhaps that not available (but a change will trigger a new p4Info from p4c downstream from those changes)

Reply:
We don't need to update the p4Info file every time there's a change in the compiler output. In fact, it's probably in our best interests to continue using the same version of the file as long as we can. We're looking for unintentional changes in behavior between two versions of the _code_.

This doesn't mean that we shouldn't have the capability to replace the p4Info file and automatically update the data the tests are sensitive too. I figured we'd write a program that reads p4Info.txt, extracts the values it needs, and writes a bunch of `constexpr`s to a header file that the test programs can `#include`.